tools/sklearn/main_macros.xml

```diff
@@ -8,6 +8,7 @@
           <requirement type="package" version="0.22.0">pandas</requirement>
           <requirement type="package" version="0.72.1">xgboost</requirement>
           <requirement type="package" version="0.9.12">asteval</requirement>
+          <requirement type="package" version="0.6">skrebate</requirement>
```

Comment:
this is not used in every tool, isn't it?

Reply:
Maybe at 2 or 3 places. I put it in the tool initially.  When debugging the pickler, I moved here to rule out the potential problem of this one. I can try to move back later. Be aware that will create a few extra lines.